spicy/toolchain/src/compiler/codegen/parser-builder.cc

```diff
@@ -1575,9 +1575,14 @@ struct ProductionVisitor : public production::Visitor {
         // Fields with stored elements should reserve the amount they will parse
         if ( auto* field = p->meta().field(); field && ! field->isTransient() ) {
             // Create a temporary so stateful size expressions are only evaluated once.
-            repeat = builder()->addTmp("repeat", repeat);
+            auto uint64 = builder()->qualifiedType(builder()->typeUnsignedInteger(64), hilti::Constness::Const);
+            repeat = builder()->addTmp("repeat", uint64, repeat);
```

Comment:
Nice!

Nit: `uint64` is a pointer, so we could use `auto*`. We are absolutely inconsistent with that though.

Reply:
As a side note, agree with using `auto*` as a general style guideline. The inconsistency is, too a large degree, because of that massive, earlier AST refactor which did a lot of search-and-replace on types. (If I were better with clang's AST tooling, I'd actually try do some automated conversion for all  `Node`-derived `auto` pointers.)

Reply:
I opened https://github.com/zeek/spicy/pull/2037 to automatically lint and fix this.